tests/integration/RequestBuilder/EventsRequestBuilderTest.php

```diff
@@ -51,4 +67,12 @@ public function shouldExecuteInteractionAndUserMergeAndItemPropertyCommands(): v
 
         $this->assertResponseCommandStatuses($response, ...$this->generateOkStatuses(10));
     }
+
+    private function buildAndSendPropertySetupRequest(ItemPropertiesSetupRequestBuilder $builder): void
+    {
+        $builder->addProperty(ItemPropertySetup::string('test_property_a'))
```

Comment:
We may add the identifier to some private class constants, to signalize these are "predefined" item properties available for use in this test. What do you thing?